ProjectCoimbra.UWP/Project.Coimbra/Helpers/SongPagesHelper.cs

```diff
@@ -114,6 +115,36 @@ public static async Task<string> UploadSongAsync()
                 return null;
             }
 
+            return await SaveFile(file);
+        }
+
+        /// <summary>
+        /// Upload Song by Dropping File Async
+        /// </summary>
+        /// <param name="e">Drag Event</param>
+        /// <returns>Uploaded File path</returns>
+        public static async Task<string> UploadSongDropAsync(DragEventArgs e)
+        {
+            if (e.DataView.Contains(StandardDataFormats.StorageItems))
+            {
+                var items = await e.DataView.GetStorageItemsAsync();
```

Comment:
Can you check if these async methods work with `.ConfigureAwait(false)` added? If so, that will make the code more efficient. If not, could you add `.ConfigureAwait(true)` so that we know this can't be used here.

Reply:
No GetStorageItemsAsync doesn't seem to have .ConfigureAwait available.
Do you mean adding .ConfigureAwait(true) onto await SaveFileAsync(file) ?

Reply:
`.ConfigureAwait(false)` should always be available on `Task` objects. In classic .NET Framework (i.e. not .NET Core), an old thread is left around for retrieving the result of the task. In many cases (but not all), this is unnecessary so you can add `.ConfigureAwait(false)` to more efficiently use resources and help the program run quicker.

I'm surprised it doesn't work here, but adding it wherever possible is good.

Reply:
Thanks for the explanation. I thought you meant to check if e.DataView.GetStorageItemsAsync() has a configureawait option. In my other PR i believe  I've set configure await to false for the UploadSongDropAsync method. I'll double check when merging